jaeger-spark-dependencies-cassandra/src/main/java/io/jaegertracing/spark/dependencies/cassandra/CassandraDependenciesJob.java

```diff
@@ -120,9 +120,15 @@ public Builder localDc(String localDc) {
       return this;
     }
 
+    public Builder properties(Map<String,String> properties){
```

Comment:
Do not expose this. Add an argument to `SpansToDependencyLinks` and propagate it there from main.